src/main/java/com/afornalik/ox/controller/PlayerController.java

```diff
@@ -0,0 +1,56 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class PlayerController {
+
+    private final UIOperations uiOperations;
+
+    PlayerController(UIOperations uiOperations) {
+        this.uiOperations = uiOperations;
+    }
+
+    String createPlayer() {
+
+        Map<String,Object> playerAttributes = new HashMap<>();
+
+        uiOperations.print("Player name : ");
+        playerAttributes.put("name",uiOperations.read());
+
+        do {
+            uiOperations.print("Select X or O mark ");
+            String tempSymbol = uiOperations.read();
+            if(tempSymbol.equalsIgnoreCase("x")){
+                playerAttributes.put("FieldStatus", FieldStatus.X);
+            }else if(tempSymbol.equalsIgnoreCase("o")){
+                playerAttributes.put("FieldStatus", FieldStatus.O);
```

Comment:
Selecting which FieldStatus should be used could be moved to some factory method in enum instead of adding if-else statement here.
Also, I can see this method is creating only one player. What would happen if you'd tried to add second one and they would select the same name and same symbol as the previous player?